web/tests/compose_validate.test.js

```diff
@@ -672,14 +674,17 @@ test_ui("warn_if_private_stream_is_linked", ({mock_template}) => {
     $("#compose_private").hide();
     compose_state.set_message_type("stream");
 
-    denmark = {
+    // Not everyone is subscribed to secret_stream in denmark, so the
+    // warning is rendered.
```

Comment:
This test was already broken on main -- if you comment out the rest of the tests in this file, it would fail. Not sure why that got revealed in this PR, but anyways I fixed it.

It would be great to run unit tests with separate state from each other.